src/bpfbytecode.cpp

```diff
@@ -34,6 +34,7 @@ static std::optional<std::string> get_global_var_section_name(
 }
 
 BpfBytecode::BpfBytecode(std::span<const std::byte> elf)
```

Comment:
Would it be simpler if the constructor took a vector arg and we just moved it into `elf_` ?

Reply:
Doesn't look like it. Looks like llvm::SmallVector doesn't easy convert to std::vector. And in aot::load() it's working with pointer + len, so span is more natural there as well. So as-is is probably simple enough